CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+# v0.3.0
+
+## Changed
+- [[#40]](https://github.com/rust-vmm/seccompiler/pull/40): Update Rust
+  to Edition 2021
+
+## Fixed
+
+- [[#31]](https://github.com/rust-vmm/seccompiler/issues/31): Implement
```

Comment:
I think this would fit better in `Added` instead of `Fixed` :-?

Reply:
I think it's better as a fix, as it made error-handling very ugly without.